src/main/java/seedu/address/ui/ReminderWindow.java

```diff
@@ -3,10 +3,12 @@
 import java.util.logging.Logger;
```

Comment:
This is very similar to `FavouriteWindow`. Could abstract out a `SingleColumnPersonListWindow`.

Reply:
thanks for pointing that out, abstracted out `FavouriteWindow` & `ReminderWindow` as `SingleColumnPersonListWindow`, removed `ReminderPersonListPanel` & `ReminderPersonCard` to use `PersonListPanel` & `PersonCard` respectively